src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -589,6 +589,52 @@ public virtual async Task PatchOneWithNonNullableFieldAsNull()
             );
         }
 
+        /// <summary>
+        /// Verifies that we throw an exception when an extraneous field provided in the request body for a PATCH operation
+        /// does not map to a valid backing column in the table.
```

Comment:
```suggestion
        /// that does not map to a valid backing column in the table.
```

Reply:
I think current wording is correct.

Reply:
Ammending the overall verbose to be more clear.